src/OpenToolkit.Mathematics/Color/Color4.cs

```diff
@@ -0,0 +1,1095 @@
+        /// </summary>
+        public static Color4<Rgba> Steelblue => new Color4<Rgba>(0.27450982f, 0.50980395f, 0.7058824f, 1f);
+
+        /// <summary>
+        /// Gets system color tan corresponding to 210 180 140 255 (0.8235294, 0.7058824, 0.54901963, 1).
+        /// </summary>
+        public static Color4<Rgba> Tan => new Color4<Rgba>(0.8235294f, 0.7058824f, 0.54901963f, 1f);
+
+        /// <summary>
+        /// Gets system color teal corresponding to 0 128 128 255 (0, 0.5019608, 0.5019608, 1).
+        /// </summary>
+        public static Color4<Rgba> Teal => new Color4<Rgba>(0f, 0.5019608f, 0.5019608f, 1f);
+
+        /// <summary>
+        /// Gets system color thistle corresponding to 216 191 216 255 (0.84705883, 0.7490196, 0.84705883, 1).
+        /// </summary>
+        public static Color4<Rgba> Thistle => new Color4<Rgba>(0.84705883f, 0.7490196f, 0.84705883f, 1f);
+
+        /// <summary>
+        /// Gets system color tomato corresponding to 255 99 71 255 (1, 0.3882353, 0.2784314, 1).
+        /// </summary>
+        public static Color4<Rgba> Tomato => new Color4<Rgba>(1f, 0.3882353f, 0.2784314f, 1f);
+
+        /// <summary>
+        /// Gets system color turquoise corresponding to 64 224 208 255 (0.2509804, 0.8784314, 0.8156863, 1).
+        /// </summary>
+        public static Color4<Rgba> Turquoise => new Color4<Rgba>(0.2509804f, 0.8784314f, 0.8156863f, 1f);
+
+        /// <summary>
+        /// Gets system color violet corresponding to 238 130 238 255 (0.93333334, 0.50980395, 0.93333334, 1).
+        /// </summary>
+        public static Color4<Rgba> Violet => new Color4<Rgba>(0.93333334f, 0.50980395f, 0.93333334f, 1f);
+
+        /// <summary>
+        /// Gets system color wheat corresponding to 245 222 179 255 (0.9607843, 0.87058824, 0.7019608, 1).
+        /// </summary>
+        public static Color4<Rgba> Wheat => new Color4<Rgba>(0.9607843f, 0.87058824f, 0.7019608f, 1f);
+
+        /// <summary>
+        /// Gets system color white corresponding to 255 255 255 255 (1, 1, 1, 1).
+        /// </summary>
+        public static Color4<Rgba> White => new Color4<Rgba>(1f, 1f, 1f, 1f);
+
+        /// <summary>
+        /// Gets system color whitesmoke corresponding to 245 245 245 255 (0.9607843, 0.9607843, 0.9607843, 1).
+        /// </summary>
+        public static Color4<Rgba> Whitesmoke => new Color4<Rgba>(0.9607843f, 0.9607843f, 0.9607843f, 1f);
+
+        /// <summary>
+        /// Gets system color yellow corresponding to 255 255 0 255 (1, 1, 0, 1).
+        /// </summary>
+        public static Color4<Rgba> Yellow => new Color4<Rgba>(1f, 1f, 0f, 1f);
+
+        /// <summary>
+        /// Gets system color yellowgreen corresponding to 154 205 50 255 (0.6039216, 0.8039216, 0.19607843, 1).
+        /// </summary>
+        public static Color4<Rgba> Yellowgreen => new Color4<Rgba>(0.6039216f, 0.8039216f, 0.19607843f, 1f);
```

Comment:
Please make them `readonly`

Reply:
Not really a reason to make this comment twice